diff --git a//etc/angie/angie.conf b//etc/angie/angie.conf
index d42a71a..32ae757 100644
--- a//etc/angie/angie.conf
+++ b//etc/angie/angie.conf
@@ -5,6 +5,30 @@ worker_rlimit_nofile 65536;
 error_log  /var/log/angie/error.log notice;
 pid        /run/angie.pid;
 
+load_module modules/ngx_http_auth_ldap_module-debug.so;
+load_module modules/ngx_http_auth_spnego_module-debug.so;
+load_module modules/ngx_http_brotli_filter_module-debug.so;
+load_module modules/ngx_http_brotli_static_module-debug.so;
+load_module modules/ngx_http_cache_purge_module-debug.so;
+load_module modules/ngx_http_dav_ext_module-debug.so;
+load_module modules/ngx_http_echo_module-debug.so;
+load_module modules/ngx_http_enhanced_memcached_module-debug.so;
+load_module modules/ngx_http_eval_module-debug.so;
+load_module modules/ngx_http_geoip2_module-debug.so;
+load_module modules/ngx_stream_geoip2_module-debug.so;
+load_module modules/ngx_http_headers_more_filter_module-debug.so;
+load_module modules/ngx_http_keyval_module-debug.so;
+load_module modules/ngx_stream_lua_module-debug.so;
+load_module modules/ngx_http_opentracing_module-debug.so;
+load_module modules/ngx_postgres_module-debug.so;
+load_module modules/ngx_http_redis2_module-debug.so;
+load_module modules/ngx_rtmp_module-debug.so;
+load_module modules/ndk_http_module-debug.so;
+load_module modules/ngx_http_set_misc_module-debug.so;
+load_module modules/ngx_http_subs_filter_module-debug.so;
+load_module modules/ngx_http_testcookie_access_module-debug.so;
+load_module modules/ngx_http_upload_module-debug.so;
+
 events {
     worker_connections  65536;
 }
